hardware_accelerators/rdma-remote-direct-memory-access.adoc

```diff
@@ -43,3 +43,19 @@ include::modules/rdma-configuring-the-nvidia-network-operator.adoc[leveloffset=+
 
 include::modules/rdma-configuring-the-gpu-operator.adoc[leveloffset=+1]
 
+// TELCODOCS-2232
+// include::modules/PREFORMATTED IMAGE.adoc[leveloffset=+1]
+
+include::modules/rdma-creating-workload-pods.adoc[leveloffset=+1]
```

Comment:
Add the new topic "Creating the machine configuration". The `machineconfig` file enables access to the pods.
Add @wabouhamad example here.

Reply:
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: worker
  name: 02-worker-container-runtime
spec:
  config:
    ignition:
      version: 3.2.0
    storage:
      files:
      - contents:
          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF91bGltaXRzID0gWwoibWVtbG9jaz0tMTotMSIKXQo=
        mode: 420
        overwrite: true
        path: /etc/crio/crio.conf.d/10-custom